config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,3 @@
+Sidekiq.configure_server do |config|
```

Comment:
Nothing they both use STDOUT, it's mainly so that any calls to Rails.logger inside a job put to the same logger. It's so that we can use Rails.logger in the LoggerAdapter

It's just a pointer reference thing basically, Sidekiq respects all of the rails initializers and can execute lib code that might use Rails.logger

In sort, I want to just use Rails.logger and have it show up in Sidekiq

Sent from my iPhone

> On Apr 18, 2016, at 2:26 PM, discorick <notifications@github.com> wrote:
> 
> In config/initializers/sidekiq.rb:
> 
> > @@ -0,0 +1,3 @@
> > +Sidekiq.configure_server do |config|
> Whats the difference between this guy and the the Rails.logger ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>